classes/Configuration.php

```diff
@@ -584,19 +584,20 @@ public static function deleteByName($key)
     }
 
     /**
-     * Delete configuration key from current context.
+     * Delete configuration key from current context
      *
      * @param string $key
+     * @param int|null $idShopGroup
+     * @param int|null $idShop
```

Comment:
```suggestion
     * @param int $idShopGroup
     * @param int $idShop
```

The only value you can pass is `int` if you want to allow null you have to do `?int`